src/interfaces.go

```diff
@@ -18,12 +19,10 @@ type Square struct {
 }
 
 func (s *Square) area() float64 {
-	l := distance(r.x1, r.y1, r.x2, r.y2) // calculate distance between 2 points
-	return l * l
+	return math.Abs(s.x2-s.x1) * math.Abs(s.y2-s.y1)
```

Comment:
I think we can have single math.Abs function wrapping the multiplication.

> math.Abs((s.x2-s.x1) * (s.y2-s.y1))

Reply:
Oh, sure! I will commit the change ASAP.